From 3312aba6ccfae7629c068dc979b39e18b2e2e838 Mon Sep 17 00:00:00 2001
From: CI Assistant <chdauto@intel.com>
Date: Thu, 9 Jan 2020 12:57:22 +0200
Subject: [PATCH] Merge pull request #1441 in SW_UGW/linux from
 bugfix/UGW_SW-44604-clear-the-flow-id-set-by-ingress-qos-module-841 to 8.4.1

* commit '804657e2e1daa7683d4d7d7b70b74bb0af5a05bf':
  UGW_SW-44604: Clearing flow id bit in RX
---
 drivers/net/datapath/dpm/gswip30/datapath_rx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/datapath/dpm/gswip30/datapath_rx.c b/drivers/net/datapath/dpm/gswip30/datapath_rx.c
index 27107b0c2987..71e9a681d3d5 100644
--- a/drivers/net/datapath/dpm/gswip30/datapath_rx.c
+++ b/drivers/net/datapath/dpm/gswip30/datapath_rx.c
@@ -245,6 +245,7 @@ int32_t dp_rx_30(struct sk_buff *skb, u32 flags)
 					desc_0, desc_1,
 					desc_2, desc_3);
 		}
+		desc_0->field.flow_id = 0;
 #if IS_ENABLED(CONFIG_INTEL_DATAPATH_MPE_FASTHOOK_TEST)
 		if (unlikely(ltq_mpe_fasthook_rx_fn))
 			ltq_mpe_fasthook_rx_fn(skb, 1, NULL);	/*with pmac */
